ray-operator/controllers/ray/common/job_test.go

```diff
@@ -87,7 +85,8 @@ func TestGetK8sJobCommand(t *testing.T) {
 		"--entrypoint-resources", strconv.Quote(`{"Custom_1": 1, "Custom_2": 5.5}`),
 		"--",
 		"echo no quote 'single quote' \"double quote\"",
-		";", "fi",
+		";", "fi", ";",
+		"ray", "job", "logs", "--address", "http://127.0.0.1:8265", "--follow", "testJobId",
```

Comment:
With this change, the submitter Job status will almost always be `Completed` even if the underlying Ray job fails, is this the behavior we want?

Reply:
Yes, I believe so. The submitter pods should be marked as `Error` only when they fail to submit the Ray job or fail to tail the Ray job logs. `spec.submitterConfig.backoffLimit` will retry these kinds of failures. Ray job failures, on the other hand, will be covered by `spec.backoffLimit`.